lahja/base.py

```diff
@@ -157,38 +158,122 @@ class BaseRemoteEndpoint(RemoteEndpointAPI):
     bi-directional with messages flowing in either direction.
     """
 
-    logger = logging.getLogger("lahja.endpoint.asyncio.RemoteEndpoint")
+    logger = logging.getLogger("lahja.base.RemoteEndpoint")
```

Comment:
While you're here maybe `AsyncioEndpoint` should also get a logger, right now I think that `self.logger` calls from inside it will use this one, which might be confusing.